src/OrleansRuntime/ReminderService/InMemoryRemindersTable.cs

```diff
@@ -18,11 +19,11 @@ internal class InMemoryRemindersTable
         // table version, as each read/insert/update should touch & depend on only one row at a time
         //internal TableVersion TableVersion;
 
-        [NonSerialized]
-        private readonly Logger logger = LogManager.GetLogger("InMemoryReminderTable", LoggerType.Runtime);
+        [NonSerialized] private readonly ILogger logger;
 
-        public InMemoryRemindersTable()
+        public InMemoryRemindersTable(ILoggerFactory loggerFactory)
```

Comment:
I believe this new version with a non-default constructor can still be serialized without issues, but can @ReubenBond validate that there will be no issues?